document/CoreFormat.mdk

```diff
@@ -668,10 +672,10 @@ the MIME content-type (RFC 2045) of the file.
 ### `region` property { #physicalLocationComponent-region }
 
 A `physicalLocationComponent` object may have a property named `region` whose value is a `region` object (&sect;[#region]),
-that represents the region within a file where the issue was detected.
+that represents the region within a file where the result occurs.
```

Comment:
I think 'result occurred' here would make this language consistent with the rest of the doc